### ISSUE TITLE
improve link selection when using accessibility options

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitica#14438]
> A new bug report from @jamiehooper:
> This is not technically a bug, but is a user interface accessibility issue for disabled people. I cannot use a mouse due to chronic pain so I use a Chrome extension called Vimium to click on links. For it to work the links have to be labeled with HTML tags. The links for checking a daily habit off are exclusively JS so I can't click on them which really limits my accessibility for website. If y'all were able to add that that would be amazing :))

**Comment by ashbork** [comment]
I think I'd like to give this a shot; I'm not sure how easy or hard it'll be (and it's my first potential contribution) but I'd appreciate being assigned to this issue
